src/main/java/com/iemr/hwc/controller/userBiomectrics/UserBiomectrics.java

```diff
@@ -1,33 +1,28 @@
-package com.iemr.hwc.controller.wo;
+package com.iemr.hwc.controller.userBiomectrics;
 
 import com.google.gson.Gson;
 import com.iemr.hwc.controller.common.master.CommonMasterController;
 import com.iemr.hwc.data.registrar.FingerPrintDTO;
 import com.iemr.hwc.data.registrar.UserBiometricsMapping;
-import com.iemr.hwc.service.location.LocationServiceImpl;
 import com.iemr.hwc.service.registrar.RegistrarServiceImpl;
 import com.iemr.hwc.utils.response.OutputResponse;
 import io.swagger.annotations.ApiOperation;
-import org.json.JSONObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.List;
-
 @CrossOrigin
 @RestController
-@RequestMapping(value = "wo/user_biometrics")
+@RequestMapping(value = "wo/user_biometrics", headers = "Authorization")
```

Comment:
You can remove 'wo' from here also.

Reply:
Change done. Please review again @Shrey475  @roopesh-beehyv